### DISCUSSION TITLE
Multiple locations

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#69] [Q&A] [answered]
Concerning multiple clinic locations with slightly different license numbers - our clinics each have a license number and different address/suite number.  example: 1234C89-K for Endocrinology, 1234C56-E for Breast clinic.  They do all use the same standard charges file.  Is anybody else interpreting this as needing a separate MRF for each clinic?

**Comment by RidenourK** [comment, marked as answer]
@sklukasatphs. Thank you for your question. 

Under § 180.50(a)(2), each hospital location operating under a single hospital license (or approval) that has a different set of standard charges than the other location(s) operating under the same hospital license (or approval) must separately make public the standard charges applicable to that location. Under § 180.50(d)(2), hospitals must ensure that the standard charge information in the MRF is “clearly identified with the hospital location with which the standard charge information is associated.” As we explained in the CY 2020 HPT Final Rule, we believed it would be sufficient for a hospital to post a single file of standard charges for a single campus location, if the file includes charges for all items and services offered at the single campus location. In cases where such off-campus and affiliated sites operate under the same license (or approval) as a main location but have different standard charges or offer different items and services, these locations would separately make public the standard charges for such locations ([84 FR 65564](https://www.federalregister.gov/citation/84-FR-65564)).

However, as indicated in the CY 2024 OPPS/ASC Final Rule, if the hospital has established a single set of standard charges for all inpatient and outpatient departments across many different locations, it could be cumbersome to list all location names and addresses in a single MRF. Therefore, CMS will require that hospitals encode the name(s) and address(es) of each hospital inpatient location and each standalone emergency department in the MRF. While strongly encouraged, it will not be required to encode all outpatient locations. We note, however, that even though we made this practical accommodation, hospitals must still include all standard charge information in the MRF, including standard charge information for outpatient locations not encoded for this data element. In other words, this accommodation should not be interpreted to mean that hospitals need not include the standard charges that apply to outpatient locations that operate under the single hospital license but whose location names and addresses are not required to be encoded. Please refer to the discussion beginning at [88 FR 82092](https://www.federalregister.gov/d/2023-24293/p-4931).

Please refer to the technical instructions for encoding “hospital location”, “hospital address” and “license number” in MRF, available in the “documentation” section of the [Hospital Price Transparency – Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency).
